src/styles/public/general.scss

```diff
@@ -210,5 +204,8 @@ main,
 	#post-content {
 		flex: 1;
 	}
+}
 
+.wp-block-image {
+	min-width: 60px;
```

Comment:
I feel like this is too broad of a change since it will affect every image on every Block Theme site. I'd rather see it made into its own class (something like `.image-min-width`) that is applied to the images you're trying to fix.